es/introduction.md

```diff
@@ -48,25 +48,25 @@ El código público está en el centro de las instituciones públicas modernas,
 
 El software público debe por tanto ser:
 
-* transparente
-* responsable
-* entendible por sus constituyentes
+* Transparente.
+* Responsable.
+* Entendible por sus constituyentes.
```

Comment:
Capital letters and punctuation like the original Standard, although I like your suggestion